### PR TITLE
Use new Localization and Downloader implementations from extractor

### PR DESCRIPTION
**Pull request by mauriciocolli**
- Make use of new Localization and Downloader implementations from extractor
- Show proper text for live stream items (watching count instead of view count)

Depends on TeamNewPipe/NewPipeExtractor#158.

### CODE REVIEW

File: app/src/main/java/org/schabi/newpipe/DownloaderImpl.java [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+
+import javax.annotation.Nonnull;
+
+import androidx.annotation.Nullable;
+import okhttp3.OkHttpClient;
+import okhttp3.RequestBody;
+import okhttp3.ResponseBody;
+
+public class DownloaderImpl extends Downloader {
+    public static final String USER_AGENT = "Mozilla/5.0 (Windows NT 6.1; WOW64; rv:43.0) Gecko/20100101 Firefox/43.0";
```
Comment (cool-student) [inline app/src/main/java/org/schabi/newpipe/DownloaderImpl.java]: Is there a reason to use such an old useragent ? I would suggest updating it (unless there's a good reason to use the old one)

```suggestion
    public static final String USER_AGENT = "Mozilla/5.0 (Windows NT 6.1; Win64; x64; rv:69.0) Gecko/20100101 Firefox/69.0";
```

File: app/src/main/java/org/schabi/newpipe/settings/ContentSettingsFragment.java [hunk 2]
```diff
@@ -109,19 +115,34 @@ public void onCreatePreferences(Bundle savedInstanceState, String rootKey) {
             return true;
         });
 
-        Preference setPreferredLanguage = findPreference(getString(R.string.content_language_key));
+        /*Preference setPreferredLanguage = findPreference(getString(R.string.content_language_key));
```
Comment (TobiGr) [inline app/src/main/java/org/schabi/newpipe/settings/ContentSettingsFragment.java]: Can you please add a comment that this is not needed due to the extractor changes and localization changes.

**Comment by mauriciocolli** [comment]
@TobiGr are you sure? [This is how the file looks like](https://github.com/TeamNewPipe/NewPipe/blob/2a15f5fb51d8455e285c0928222b4e5c55a480d3/app/src/main/res/values/settings_keys.xml#L403..L651) as of the last commit.

It seems that currently only the language names are done that way.

**Comment by TobiGr** [comment]
My bad, I mixed content language and country during my review.

**Comment by mauriciocolli** [comment]
@TobiGr Done.

**Pull request merged**